### ISSUE TITLE
Plugin Update on 3.0.2  / Import of Payment methods

### ISSUE THREAD

**Issue by jensorben** [Adyen/adyen-shopware5#194]
**Describe the bug**
We updated the plugin from Version 2.0.0 to 2.0.2. We start to to import adyen payment methods to get the native Payment methods in the shop. 
Obviously the plugin imported payment methods from the adyen test enviroment (Screenshot 0) although the plugin is configured for LIVE Mode. And the import wasn't complete at all - CarteBancaire and Visadankort e.g. are missing! 
We tried to reinstall the plugin and made a full uninstall / new install afterwards. With new entered credentials and new import of payment methods we had much more trouble:

- Several payment methods were listed twice (Screenshot 1)
- The checkout page won't work any more (Screenshot 2)
 
**To Reproduce**
1. Go to 'Settings' => Adyen Shopware Plugin 
2. Set Enviroment to LIVE-Mode
3. Import Adyen payment methods
4. Pop-Up says "33 Payment methods successfully imported " although in our LIVE-System there are only 8 activated
5. With an uninstall and reinstall and a 2nd import - which uses Test System once more - most Payment mehtods are displayed twice! (Screenshot 1)
6. After second install, the checkout process won't work any more. (Screenshot 2)

**Expected behavior**
We expected, that only 8 methods are imported! After uninstall all Adyen aprts should be cleaned!

**Screenshots**
Screenshot 0: In Adyen Test System configured payment methods
Screenshot 1: Mostly all payment methods were displayed twice after reinstall. Carte bancaire and Visadankort are missing at all.
Screenshot 2: Error on checkout page after reinstall.

What went wrong?

Best 

Jens

**Comment by NilsVankan** [comment]
Could you possible send us the Adyen log file so we can investigate the issue further?
And i don't see any links to the screenshots can you deliver us these aswell?

Kind regards,

Nils

**Comment by jensorben** [comment]
Hello Nils,
sorry for my late reply, but I felt ill for a couple of days...

Here are the screenshots.

![Screenshot2](https://user-images.githubusercontent.com/69344793/143466398-3e787eb2-eed1-4814-a23d-9732a8e55015.jpg)
![Screenshot1](https://user-images.githubusercontent.com/69344793/143466402-67edab72-2ba6-4d18-bc74-02e169e1fb81.jpg)
![Screenshot0](https://user-images.githubusercontent.com/69344793/143466407-990c839e-dd22-4717-9918-5d358759b87b.jpg)

We reset our test system in the Meantime but I will try to get some informations from the backup.

Kind regards

Jens

**Comment by jensorben** [comment]
Hello Nils,
here are the log files from this date!

Best
[logs-adyen-installation.zip](https://github.com/Adyen/adyen-shopware5/files/7604546/logs-adyen-installation.zip)

Jens

**Comment by NilsVankan** [comment]
Hey Jens,
Thanks for the extra information.
We are looking in to it to resolve it.

Kind regards

**Comment by NilsVankan** [comment]
Hey jensorben,
This issue is fixed in the 3.2.0 version.
Could you be able to update your version.
If you still encounter issues then just give a sign

Kind regards

**Comment by jensorben** [comment]
Hello Nils,
we updated our shop to V5.7.7 to install your version 3.2.0! 
We could import the payment methods from the adyen live enviroment, this works, but afterwards we got several problems on the checkout front page:
The page won't load the payment methods throwing an error while trying to load Apple Pay (which isn't active in our LIVE-Enviroment). [Screenshot 1]
After outlining this part, we got another error [Screenshot 2]
We figured out the Payment Methods were loaded nevertheless but had been hidden by css-Class "adyen-hidden--all".
And finally the input fields for credit card data are missing (and the fields for formerly saved card data,too)

Best

Jens
![Screenshot 1 2022-01-13 155019](https://user-images.githubusercontent.com/69344793/149357160-9d6ccb89-ad14-4c74-a398-74a972dfbdea.jpg)
![Screenshot 2 2022-01-13 160023](https://user-images.githubusercontent.com/69344793/149357164-da40c5a3-7962-431b-be40-a42cf1891e06.jpg)
